reporters_db/data/reporters.json

```diff
@@ -9531,6 +9570,23 @@
             }
         }
     ],
+    "N.Y. Super. Ct.": [
+        {
+            "cite_type": "state",
+            "editions": {
+                "N.Y. Super. Ct.": {
+                    "end": null,
+                    "start": "1750-01-01T00:00:00"
+                }
+            },
+            "mlz_jurisdiction": [
+                "us:ny;superior.court"
+            ],
+            "name": "Reports of Cases Argued and Determined in the Superior Court of the City of New York",
+            "notes": "See: https://catalog.hathitrust.org/Record/008606745",
```

Comment:
This might be a bit better in the `href` field instead of the notes field. I'm not sure it matters that much, but we have it, so I suppose we should use it.